src/Umbraco.Web/umbraco.presentation/umbraco/members/EditMemberGroup.aspx.cs

```diff
@@ -67,7 +67,7 @@ private void save_click(object sender, EventArgs e)
 			_memberGroup.Text = NameTxt.Text;
             memberGroupName.Value = NameTxt.Text;
             _memberGroup.Save();
-            this.ClientTools.ShowSpeechBubble(speechBubbleIcon.save, ui.Text("speechBubbles", "editMemberGroupSaved", base.getUser()),"");

```

Comment:
This looks like a leftover from your other PR. Could you revert this?

Reply:
Oh dear - yep. One sec...